2024/calcite-react/demo/src/Components/Panel/Panel.tsx

```diff
@@ -51,15 +40,24 @@ export const Panel = ({ view, panelHeading }: PanelProps) => {
         });
       },
     );
-
-    return () => clickHandle.remove();
+    return () => {
+      featuresWidget?.destroy();
```

Comment:
hmm, I am not sure if you need to call `featuresWidget?.destroy();` or whether it would be automatic
did you read about the destroy method somewhere in the docs?
I don't see it in https://qawebgis.esri.com/arcgis-components/external-documentation/?path=/docs/map-components_component-reference-features--docs

Reply:
Not sure, I had just assumed but since it's not in the docs I will leave it out for now.